Resources/Private/JavaScript/Components/Frame/index.js

```diff
@@ -0,0 +1,109 @@
+// 98997e98ea6d5ececc49db0de528151e40e272b7 from https://github.com/ryanseddon/react-frame-component/commit/98997e98ea6d5ececc49db0de528151e40e272b7
```

Comment:
Ah, so you just copy pasted the whole component from somewhere else? Why not include it via npm?